src/main/java/org/truffleruby/parser/Translator.java

```diff
@@ -60,6 +60,7 @@
         m.put("$INPUT_RECORD_SEPARATOR", "$/");
         m.put("$>", "$stdout");
         m.put("$PROGRAM_NAME", "$0");
+        m.put("$CHILD_STATUS", "$?");
```

Comment:
Yeah, I was just wondering if it would just work now that we have a more faithful globals implementation.